### PR TITLE
[WFLY-14148]: WFLY-14148 Remove MicroProfile Opentracing from WildFlyEE

### PR DESCRIPTION
**Pull request by ehsavoie**
Jira: https://issues.redhat.com/browse/WFLY-14148

### CODE REVIEW

**Comment by msve** [comment]
ack for the analysis from QE

File: microprofile/WFLY-14148-microprofile-opentracing-move.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+=== Affected Projects or Components:
+
+* WildFly
+
+== Requirements
+
+
+* The MicroProfile Opentracing functionality is provided in the full 'wildfly' feature pack only
+* Adding a new 'microprofile-opentracing' layer in the microprofile galleon pack and adding it to the microprofile 'observability' layer.
```
Comment (bstansberry) [inline microprofile/WFLY-14148-microprofile-opentracing-move.adoc:40]: s/microprofile galleon pack/'wildfly' feature pack/g
s/microprofile/'wildfly' feature pack's/g

File: microprofile/WFLY-14148-microprofile-opentracing-move.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* Removing it from wildfly ee feature pack 'observability' layer.
```
Comment (bstansberry) [inline microprofile/WFLY-14148-microprofile-opentracing-move.adoc:41]: s/wildfly ee/'wildfly-ee'/g

File: microprofile/WFLY-14148-microprofile-opentracing-move.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* Adding a feature_group to cover the default configuration with a Jaeger tracer.
+
```
Comment (bstansberry) [inline microprofile/WFLY-14148-microprofile-opentracing-move.adoc:43]: Please add that testsuite modules outside of the 'testsuite/integration/microprofile' and 'testsuite/integration/microprofile-tck' will not require MicroProfile OpenTracing to be present to run. Any tests that do will be moved to testsuite/integration/microprofile.

**Review by bstansberry — CHANGES_REQUESTED** [review]

**Pull request merged**